home-style.css

```diff
@@ -0,0 +1,467 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: grid;
+  height: 100vh;
+  grid-template-rows: 60px 1fr ;
+  grid-template-columns:1fr minmax(0,2fr) minmax(2fr,5fr) minmax(0,1fr) 1fr;
+  grid-template-areas: 
+    '. logo header searchHeader .'
+    '. left-side-menu main-content right-side-bar .'
+  ;
+  background-color:
+   black;
+}
+
+.logo {
+  grid-area: logo;
+  background-color: #000000;
+
+  svg {
+    padding: 10px;
+    height: 40px;
+    width: 40px;
+    fill: white;
+  }
+
+}
```

Comment:
You can't nest CSS selectors inside of other selectors, you'd need to use specificity:

```css
.logo {
  grid-area: logo;
  background-color: #000000;
}

.logo svg {
    padding: 10px;
    height: 40px;
    width: 40px;
    fill: white;
}
```